jobs/european-gravitational-observatory-computer-scientist-software-developer-for-multi-messenger-astronomy.html

```diff
@@ -0,0 +1,78 @@
+---
+title: Computer Scientist / Software Developer for multi-messenger astronomy
+company: European Gravitational Observatory
+url: https://www.ego-gw.it/blog/2020/08/09/computer-scientist-software-developer-for-multi-messenger-astronomy/
+location: Cascina (Pisa), Tuscany, Italy
+contract: permanent
+contact:
+    name: EGO Jobs
+    email: jobs@ego-gw.it
+created: !!timestamp 2018-04-25
```

Comment:
Should this be updated to a more recent timestamp?

Reply:
Done